plugins/UM3NetworkPrinting/src/Network/LocalClusterOutputDevice.py

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2019 Ultimaker B.V.
+# Copyright (c) 2022 Ultimaker B.V.
```

Comment:
Nothing changed in this file, why update the Copyright?

Reply:
Probably a mistake because I originally changed something, then undid my changes but forgot to undo my changes to the copyright header.

I'll change it to 2020 which was the last actual change.